src/process.ts

```diff
@@ -28,17 +28,22 @@ export class ChildProcessWrapper {
       return undefined;
     }
     if (isWin32) {
-
       const res = cp.execSync(
         `(Get-WmiObject Win32_Process -Filter "ParentProcessID=${ppid}" | Where CommandLine -like '*build_runner watch*').ProcessId`,
         { encoding: "utf8", shell: "powershell.exe" }
       );
       console.log(parseInt(res));
       return res;
-    } else {
-      const res = this.execSync(`ps xao pid,ppid | grep "\d* ${ppid}"`);
-      return res.split(' ')[0];
+    } else if (isLinux) {
+      const res = this.execSync(`ps xao pid,ppid | grep -E "[[:digit:]]+[ ]+${ppid}"`);
```

Comment:
In trying to fix this, I found out that grep regexes are very strange, this works on linux but not on macos, however as noted below macos does not show any child PIDs for the build_runner process.

Reply:
Are you sure bout it ? I'm pretty sure I remember being able to obtain a pid. Will look into that.